Source/WebCore/rendering/RenderObject.h

```diff
@@ -821,6 +821,7 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr<RenderOb
 
     // Convert the given local point to absolute coordinates. If OptionSet<MapCoordinatesMode> includes UseTransforms, take transforms into account.
     WEBCORE_EXPORT FloatPoint localToAbsolute(const FloatPoint& localPoint = FloatPoint(), OptionSet<MapCoordinatesMode> = { }, bool* wasFixed = nullptr) const;
+    std::unique_ptr<TransformationMatrix> localToAbsoluteTransform() const;
```

Comment:
Until we know better, I'd like this to have a name which makes it less likely that other people will find it and use it. Maybe the name should be specific to view transitions.